src/app/clusters/thermostat-server/thermostat-server-events.h

```diff
@@ -0,0 +1,56 @@
+/**
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/****************************************************************************
+ * @file
+ * @brief APIs for sending events from the Thermostat cluster.
+ *
+ *******************************************************************************
+ ******************************************************************************/
+
+#pragma once
+
+void EmitSystemModeChangeEvent(chip::EndpointId endpoint,
```

Comment:
Why is this a global function not namespaced in any way?

At some point here, we really need to have themostat server instance objects, with these things becoming instance methods.  But for now at least they should not be in the global namespace.  Should be namespaced under `chip::app::Clusters::Thermostat` or something?